### ISSUE TITLE
Plugin breaks complex values when `light-dark` is nested

### ISSUE THREAD

**Issue by romainmenke** [postcss/postcss-dark-theme-class#30]
```css
.light-dark-function-mix-a {
	color: light-dark(color-mix(in oklch, red, light-dark(cyan, rgb(0, 0, 0))), blue);
}
```

becomes

```css
@media (prefers-color-scheme:dark) {

	:where(html:not(.is-light)) .light-dark-function-mix-a {
		color: blue;
	}
}

:where(html.is-dark) .light-dark-function-mix-a {
	color: blue;
}

@media (prefers-color-scheme:light) {

	:where(html:not(.is-dark)) .light-dark-function-mix-a {
		color: color-mix(in oklch, red, cyan;
	}
}

:where(html.is-light) .light-dark-function-mix-a {
	color: color-mix(in oklch, red, cyan;
}
```

`color: color-mix(in oklch, red, cyan;`

-----

Might be better to use a value parser for the value replacements :)

But as the title suggests, these are complex values that are unlikely to be used by authors.

**Comment by ai** [comment]
Can I ask you to do it? I am too busy on another project right now.

**Comment by romainmenke** [comment]
Equally too busy and not blocking for me personally :)
This only surfaced because I was noticed the sourcemaps issue.

You can just leave the issue open.
If someone needs it they can fix it and submit a patch.

**Comment by ai** [comment]
@VladBrok maybe you have a time for fix `light-dark()` by moving to `postcss-value-parser`?

**Comment by ai** [comment]
The both fixes was released in 1.2.3. Thanks to everyone.

**Comment by romainmenke** [comment]
Awesome!
Super speedy fix @VladBrok 🚀
